src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/PortableInfoboxObject.java

```diff
@@ -275,7 +315,7 @@ public PortableInfoboxObject areUnorderedListAndDataValuesMarginEqual() {
         unorderedLists.get(0).getCssValue("margin"),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "margin" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)